magic/magic1.cpp

```diff
@@ -78,6 +78,38 @@ void spellShortcut(char *spell) {
         strcpy(spell, "detect-invis");
     if(!strcasecmp(spell, "d-m"))
         strcpy(spell, "detect-magic");
+
+    if(!strcasecmp(spell, "mm"))
```

Comment:
gross

Reply:
With these many, it's probably better to look into either a switch statement, an else if chain, a return after each, or a proper lookup.  

If you c m --> it's still going to check every other single one, and that's just silly.

else if chain or return is probably the easiest for now, a proper map based lookup (and redoing *set) is the better long term option